docs/commands/push.md

```diff
@@ -90,6 +90,8 @@ redocly push [-u] [--batch-id id] [--batch-size number] <path/to/definition.yaml
 | --upsert, -u     | boolean  | Upsert an API to the API registry. See [the Upsert an API with push section](#upsert-an-api-with-push) for more information.                                                                                                                                                                                                                |
 | --version        | boolean  | Show version number.                                                                                                                                                                                                                                                                                                                        |
 | --files          | [string] | List of other folders and files to upload. See [the Files section](#files) for more information.                                                                                                                                                                                                                                            |
+| --config         |  string  | Specify path to the [config file](../configuration/index.mdx).                                                                                                                                                                                                                                                                              |
```

Comment:
Reverting back due to changes in https://github.com/Redocly/redocly-cli/pull/1137
```suggestion

```